packages/avatars/src/components/Avatar.spec.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render } from 'garden-test-utils';
+import { getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import Avatar from './Avatar';
+
+const activeBoxShadow = DEFAULT_THEME.shadows.sm(getColor('crimson', 400));
+
+describe('Avatar', () => {
+  it('passes ref to underlying DOM element', () => {
+    const ref = React.createRef();
+    const { container } = render(
+      <Avatar ref={ref}>
+        <img alt="" />
+      </Avatar>
+    );
+
+    expect(container.firstChild).toBe(ref.current);
```

Comment:
I'm simply proving that `forwardRef` is wrapping the `StyledAvatar`. In some of our refactors, this has slipped through the cracks, so I put a test to prevent that from happening in the future.